preparation/acoustic_preparation.ipynb

```diff
@@ -1228,7 +1228,15 @@
     "    'num_ckpt_keep': num_ckpt_keep,\n",
     "    'max_updates': max_updates,\n",
     "    'permanent_ckpt_start': permanent_ckpt_start,\n",
-    "    'permanent_ckpt_interval': permanent_ckpt_interval\n",
+    "    'permanent_ckpt_interval': permanent_ckpt_interval,\n",
+    "    \n",
+    "    ###########\n",
+    "    # pytorch lightning\n",
+    "    # Read https://lightning.ai/docs/pytorch/stable/common/trainer.html#trainer-class-api for possible values\n",
+    "    ###########\n",
+    "    'pl_trainer_accelerator': 'auto',\n",
+    "    'pl_trainer_devices': 'auto',\n",
```

Comment:
All configuration keys should be added to existing config files (in this case, base.yaml). Configurations of each dataset overrides their values, but they should have no unique keys to prevent KeyError when training with config files from the old branches or versions.